aptible/resource_environment.go

```diff
@@ -0,0 +1,135 @@
+package aptible
+
+import (
+	"context"
+	"github.com/aptible/go-deploy/aptible"
+	"log"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceEnvironment() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceEnvironmentCreate,
+		ReadContext:   resourceEnvironmentRead,
+		UpdateContext: resourceEnvironmentUpdate,
+		DeleteContext: resourceEnvironmentDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceEnvironmentImport,
+		},
+		Schema: map[string]*schema.Schema{
+			"env_id": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+			"org_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```

Comment:
There may be a UUID validation function provided by terraform.

Reply:
i can, but it'll just pipe up a 400-type if invalid as well (since it's passed around as strings) in practice. would be a nice validation though good point